Sources/Sentry/include/HybridPublic/SentryEnvelopeItemType.h

```diff
@@ -7,3 +7,4 @@ static NSString *const SentryEnvelopeItemTypeClientReport = @"client_report";
 static NSString *const SentryEnvelopeItemTypeProfile = @"profile";
 static NSString *const SentryEnvelopeItemTypeReplayVideo = @"replay_video";
 static NSString *const SentryEnvelopeItemTypeStatsd = @"statsd";
+static NSString *const SentryEnvelopeItemTypeProfileChunk = @"profile_chunk";
```

Comment:
`h`: We must add a data category related to this envelope item type or map this envelope item type to the data category profile so rate limiting and client reports work correctly. See https://github.com/getsentry/sentry-cocoa/blob/e072ad1b27ff5e6413edd2e4716a5ef0be4eee04/Sources/Sentry/SentryDataCategoryMapper.m#L35-L37